spec/features/donuts/02_user_add_new_donut_spec.rb

```diff
@@ -11,41 +11,58 @@
   # [ ] On donut#new there should be a Form
   # [ ] Submitting correct form takes us to donut#show
   # [ ] Submitting incorrect renders the form with errors
-  before(:each) do
-    visit root_path
-  end
 
-  scenario "User see a new donut form with correct inputs" do
-    click_link "Add New Donut"
-    expect(page).to have_css("input#donut_name")
-    expect(page).to have_css("input#donut_vendor_name")
-    expect(page).to have_css("input#donut_image")
-    expect(page).to have_css("textarea#donut_description")
-    expect(page).to have_css("input#donut_user_id")
+  context "as an unauthenticated user" do
+    scenario "I will not be able to add a new donut" do
+      visit root_path
+
+      expect(page).to_not have_content("Add New Donut")
+    end
   end
 
-  scenario "User fills in new donut form inputs sucessfully!" do
-    click_link "Add New Donut"
-    fill_in("Name", with: "glazed")
-    fill_in("Vendor name", with: "best")
-    fill_in("Image", with: "https://goo.gl/dfV24M")
-    fill_in("Description", with: "Everyone loves this donut.")
-    fill_in("User", with: 1)
+  context "as an authenticated user" do
+    let(:user) { create(:user) }
 
-    click_button("Create Donut")
+    before(:each) do
+      login_as(user)
+      visit root_path
+      click_link "Add New Donut"
+    end
+    scenario "I can successfully add a new donut by navigating to the root path
+      and clicking the Add New Donut link" do
 
-    expect(page).to have_content("glazed")
-  end
+      expect(page).to have_css("input#donut_name")
+      expect(page).to have_css("input#donut_vendor_name")
+      expect(page).to have_css("input#donut_image")
+      expect(page).to have_css("textarea#donut_description")
+      expect(page).to have_css("input#donut_user_id")
+    end
+
+    scenario "I can successfully add a new donut by filling form with valid
+      inputs" do
+      fill_in("Name", with: "glazed")
+      fill_in("Vendor name", with: "best")
+      fill_in("Image", with: "https://goo.gl/dfV24M")
+      fill_in("Description", with: "Everyone loves this donut.")
+      fill_in("User", with: 1)
+
+      click_button("Create Donut")
+
+      expect(page).to have_content("glazed")
+    end
+
+    scenario "I will not be able to add a donut if I submit the new donut form
```

Comment:
Tweak this assertion. It should be one sentence. You can break it down by nesting it further. For example you have `as an authorized user` then `using valid data to create a donut` `I see a success message upon submission` or `using invalid values to create a donut` `the page rerenders with an error message`